igvc_gazebo/nodes/control/main.cpp

```diff
@@ -59,13 +55,37 @@ int main(int argc, char **argv)
   ros::init(argc, argv, "jaymii_controller");
 
   ros::NodeHandle handle;
+  ros::NodeHandle pNh("~");
 
   ros::Publisher leftWheelEffortPublisher =
       handle.advertise<std_msgs::Float64>("/left_wheel_effort_controller/command", 1);
   ros::Publisher rightWheelEffortPublisher =
```

Comment:
change to `right_wheel_effort_publisher`.

> In general, variable names are under_scored.

http://wiki.ros.org/CppStyleGuide#Variables